metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/UpdateIndicesHook.java

```diff
@@ -198,15 +199,13 @@ private void updateSearchService(String entityName, Urn urn, AspectSpec aspectSp
       return;
     }
 
-    String docId;
-    try {
-      docId = URLEncoder.encode(urn.toString(), "UTF-8");
-    } catch (UnsupportedEncodingException e) {
-      log.error("Failed to encode the urn with error: {}", e.toString());
+    Optional<String> docId = SearchUtils.getDocId(urn);
```

Comment:
Why wouldn't doc id be present? bad urn?

Reply:
I am suspecting some character that cannot be url encoded. The urlencoder throws this exception